library/core/class.dispatcher.php

```diff
@@ -391,7 +391,7 @@ private function analyzeRequest($request) {
         if ($controllerName) {
             // The controller was found based on the path.
             $result['controller'] = $controllerName;
-        } elseif (Gdn::pluginManager()->hasNewMethod('RootController', val(0, $parts))) {
+        } elseif (Gdn::pluginManager()->hasNewMethod('RootController', $parts[0]??false)) {
```

Comment:
https://docs.vanillaforums.com/developer/contributing/coding-standard-php/#1-overview

Should be: `$parts[0] ?? false`

Reply:
Sorry, updated...